cluster/examples/kubernetes/ceph/direct-mount.yaml

```diff
@@ -44,8 +44,6 @@ spec:
             name: libmodules
           - name: mon-endpoint-volume
             mountPath: /etc/rook
```

Comment:
This needs a new ceph release right? as the base image also need to have this RBD CLI option